src/graphics/render-target.js

```diff
@@ -238,3 +258,18 @@ pc.extend(pc, function () {
         RenderTarget: RenderTarget
     };
 }());
+
+/**
+ * @function
+ * @name pc.RenderTarget.copy
+ * @description Copies color and/or depth contents of one render target to another. Formats, sizes and anti-aliasing samples must match.
+ * Depth buffer can only be copied on WebGL 2.0.
+ * @param {pc.RenderTarget} source Source render target to copy from. Can be null for back buffer.
+ * @param {pc.RenderTarget} dest Destination render target to copy to. Can be null for back buffer.
+ * @param {Boolean} color Copy color buffer
+ * @param {Boolean} depth Copy depth buffer
+ */
+pc.RenderTarget.copy = function(source, dest, color, depth) {
```

Comment:
But there is same method on device `copyRenderTarget`, one of them is redundant as public API.